recipes/tinycbor/0002-Fix-static_assert-compiler-support.patch

```diff
@@ -0,0 +1,25 @@
+From 6639f6db4d0a0b88dbbd7c0eda6a2b1e08ae22d2 Mon Sep 17 00:00:00 2001
+From: Samuel Debionne <samuel.debionne@esrf.fr>
+Date: Wed, 23 Aug 2023 16:14:19 +0200
+Subject: [PATCH 2/2] Fix static_assert compiler support
+
+---
+ src/compilersupport_p.h | 2 +-
```

Comment:
Have you opened a pull request to upstream this patch?

Reply:
I have just opened https://github.com/intel/tinycbor/pull/242